### DISCUSSION TITLE
Is it possible to programatically open the file picker when using FileUploader

### DISCUSSION THREAD

**Discussion by malimccalla** [segmentio/evergreen#1650] [Q&A]
I need the file picker dialogue to appear when `FileUploader` is rendered for the first time. The underlying input element doesn't appear to have a consistent ID I can use to do this. I have also tried adding a react ref to the component but having no luck with that either

Thanks!

**Comment by brandongregoryscott** [comment]
I don't think this is possible or something we're really interested in exposing the ability to do. The user should have to trigger the native file picker dialog with an action (click, enter or space key when focused) - do you have a CodeSandbox that can demonstrate the use-case for this UX?

**Reply by malimccalla** [reply]
Unfortunately no code sandbox but I'm attempting to create a UX similar to that of LinkedIn and other social media sites. Similar to the LinkedIn example below my filepicker is displayed in a Dialog, and would be great to open the picker as the component renders, removing the need to another click


https://github.com/segmentio/evergreen/assets/18899991/bc401b32-577c-49cf-91ed-cec4281bf36a
